tests/backstop_data/engine_scripts/puppet/mosaico/mailing/reciepents-modal.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./step-1.js') (engine, scenario, viewport);
+  await engine.waitFor(() => document.querySelector('a[title="Preview a List of Recipients"]').innerHTML !== 'Estimating...');
```

Comment:
isn't there a selector that we can `waitFor` instead

Reply:
Unfortunately. Angular just changes the text of the `<a>` tag using scope variable binding and there is no dom event apart from it on which we can wait for.